ape_safe/_cli/modules.py

```diff
@@ -0,0 +1,67 @@
+import click
+from ape.cli import ConnectedProviderCommand, account_option, ape_cli_context, network_option
+from ape.types import AddressType
+
+from ape_safe._cli.click_ext import safe_argument
+
+
+@click.group()
+def modules():
+    """
+    Commands for handling safe modules
+    """
+
+
+@modules.command("list", cls=ConnectedProviderCommand)
+@network_option()
+@safe_argument
+def _list(safe):
+    """List all modules enabled for SAFE"""
+    for module in safe.modules:
+        click.echo(repr(module))
+
+
+@modules.command(cls=ConnectedProviderCommand)
+@network_option()
+@safe_argument
+def guard(safe):
+    """Show module guard (if enabled) for SAFE"""
+    if guard := safe.modules.guard:
+        click.echo(f"Guard: {guard}")
+
+    else:
+        click.secho("No Module Guard set", fg="red")
+
+
+@modules.command(cls=ConnectedProviderCommand)
+@ape_cli_context()
+@network_option()
+@account_option()
+@safe_argument
+@click.option("--propose", is_flag=True, default=False)
+@click.argument("module")
+def enable(cli_ctx, safe, account, module, propose):
+    """
+    Enable MODULE for SAFE
+
+    **WARNING**: This is a potentially destructive action, and may make your safe vulnerable.
+    """
+    module = cli_ctx.conversion_manager.convert(module, AddressType)
+    safe.modules.enable(module, submitter=account, propose=propose)
+
+
+@modules.command(cls=ConnectedProviderCommand)
+@ape_cli_context()
+@network_option()
+@account_option()
+@safe_argument
+@click.option("--propose", is_flag=True, default=False)
+@click.argument("module")
+def disable(cli_ctx, safe, account, module, propose):
+    """
+    Disable MODULE for SAFE
+
+    **WARNING**: This is a potentially destructive action, and may impact operations of your safe.
```

Comment:
does this warning show up nice in the cli doc?

Reply:
We don't do any fancy highlighting (using `rich` or w/e), but I think this looks decent
```sh
$ ape safe modules disable -h
Usage: ape safe modules disable [OPTIONS] SAFE MODULE

  Disable MODULE for SAFE

  **WARNING**: This is a potentially destructive action, and may impact
  operations of your safe.

Options:
  -v, --verbosity LVL             One of ERROR, WARNING, SUCCESS, INFO, or
                                  DEBUG
  --account ACCOUNT
  --propose
  --network [ecosystem-name][:[network-name][:[provider-name]]]
                                  Override the default network and provider.
                                  (see `ape networks list` for options)
  -h, --help                      Show this message and exit.
```